docs/source/dev/sphinx.rst

```diff
@@ -39,6 +39,20 @@ The following requirements need to be installed (once) to build our documentatio
     # python tools & style theme
     pip install -r requirements.txt # --user
 
+In order to not break any of your existing Python configurations, you can also create a new environment that you only use for building the documentation.
+Since it is possible to install doxygen with conda, the following demonstrates this.
+
+.. code-block:: bash
+
+    cd docs/
+
+    # create a bare conda environment containing just all the requirements
+    # for building the picongpu documentation
+    # note: also installs doxygen inside this environment
+    conda env create picongpu-docs-env.yml
```

Comment:
when I run thi scommand, I get the following error:
```bash
SpecNotFound: Invalid name, try the format: user/package
```

Reply:
Might be darwin (MacOS) related issue.

Reply:
I am using `conda 4.4.10`

Reply:
Haha, sorry, the `--file` was missing which is correct in the file comment itself 
Good find!